GliaWidgets/Component/OnHoldOverlay/OnHoldOverlayView.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+import PureLayout
+
+final class OnHoldOverlayView: UIView {
+    private let style: OnHoldOverlayStyle
+    private let imageSize: CGSize
+    private let blurEffectView = OnHoldOverlayVisualEffectView()
+    private let imageView = UIImageView()
+
+    init(
+        style: OnHoldOverlayStyle,
+        imageSize: CGSize
+    ) {
+        self.style = style
+        self.imageSize = imageSize
+
+        super.init(frame: .zero)
+
+        setup()
+        layout()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setup() {
+        imageView.image = style.image.withRenderingMode(.alwaysTemplate)
+        imageView.tintColor = style.imageColor
+    }
+
+    private func layout() {
+        addSubview(blurEffectView)
+        blurEffectView.autoPinEdgesToSuperviewEdges()
```

Comment:
perhaps I miss something, but for me it looks like `blurEffectView` will be placed under the imageView and I don't see any re-ordering here. What is the benefit of this `blurEffectView` view?

Reply:
imageView is for the pause icon asset. The whole overlay view will be placed on top of the user image view, which will blur the user image, but not the pause icon asset on top of it